### ISSUE TITLE
[Bug]: Undocumented attribute loading="lazy" added to all images

### ISSUE THREAD

**Issue by John5** [pimcore/pimcore#11305]
### Expected behavior

When using the HTML output for a thumbnail I expect no additional attributes to be added. Any attributes that are added should be documented and allowed to be disabled.

When embedding an thumbnail image in a template like this:

```
1. {{ mainImage.thumbnail('my-thumb').html }} 
2. {{ mainImage.thumbnail('my-thumb').html({'loading': 'eager'}) }}
3.  {{ mainImage.thumbnail('my-thumb').html({'loading': 'lazy'}) }}
```

The first two images should not have a loading="lazy" as an attribute. For the first one because the documentation does not mention this and the second one should not have it because it is explicitly overridden.

### Actual behavior

loading="lazy" attribute is always added to all <img> elements.

### Steps to reproduce

Add thumbnail image in twig template and observe HTML output.

**Comment by kingjia90** [comment]
I've took a look, it seems that the option "[loading](https://pimcore.com/docs/pimcore/current/Development_Documentation/Assets/Working_with_Thumbnails/Image_Thumbnails.html#page_Generating-HTML-for-Thumbnails)" isn't available in the method and wouldn't be considered ([unlike alt and title](https://github.com/pimcore/pimcore/blob/10df71ba81014d999d330d2bf28c2f946a1dde54/models/Asset/Image/Thumbnail.php#L377)), but it can be passed through the imgAttributes array.
In other words, it should look like `mainImage.thumbnail('my-thumb').html({'imgAttributes': {'loading': 'eager'}})`.

It doesn't seem a bug, but surely could improved. Need to specify somewhere in the documentation that lazy loading is the [default](https://github.com/pimcore/pimcore/blob/10df71ba81014d999d330d2bf28c2f946a1dde54/models/Asset/Image/Thumbnail.php#L412).
